src/Middleware/WorkerMiddleware.php

```diff
@@ -102,6 +102,28 @@ private function getMiddlewareForMessage(string $messageName): callable
             ));
         }
 
-        return $this->container->get($this->messagesMapping[$messageName]);
+        if (is_string($this->messagesMapping[$messageName])) {
+            return $this->container->get($this->messagesMapping[$messageName]);
+        }
+
+        if (!is_array($this->messagesMapping[$messageName])) {
+            // throw exception: must be array or string
+        }
+
+        // Instantiate middlewares
+        $middlewares = array_map([$this->container, 'get'], $this->messagesMapping[$messageName]);
+
+        // Build and return a simle pipeline
+        $pipeline = function ($req, $res, $out) use ($middlewares, &$pipeline) {
```

Comment:
Because this variable is not defined at this moment (we're declaring `$pipeline` in same line).